### DISCUSSION TITLE
32-bit FLAC support

### DISCUSSION THREAD

**Discussion by TrixieDev** [libsndfile/libsndfile#910] [Ideas]
As of [libflac version 1.4.0](https://github.com/xiph/flac/releases/tag/1.4.0) (released in September 2022), the library supports encoding and decoding of 32-bit FLAC streams. I wonder if it'd be possible to enable 32-bit support for FLAC in sndfile any time soon(ish)?

**Comment by evpobr** [comment]
Hi @TrixieDev . Good idea. Maybe @arthurt can take this, if he has time.

**Comment by ctem** [comment]
This remains the only Idea-category discussion ever posted to this repo. I went ahead and opened an issue for it so it can be tagged (e.g., `enhancement` and/or `wishlist`) and tracked like all the other feature requests.

**Comment by robertbcalhoun** [comment]
See issue #980 .
